editor/src/core/property-controls/property-controls-local.ts

```diff
@@ -173,6 +178,21 @@ export function createModuleEvaluator(editor: EditorState): ModuleEvaluator {
 export const isComponentDescriptorFile = (filename: string) =>
   filename.startsWith('/utopia/') && filename.endsWith('.utopia.js')
 
+function isComponentRegistrationValid(
+  registrationKey: string,
+  registration: ComponentToRegister,
+): boolean {
+  if (typeof registration.component === 'undefined') {
+    return false
+  }
+
+  if (isComponentRendererComponent(registration.component)) {
+    return registration.component.originalName === registrationKey
```

Comment:
this will need to be re-written if we want to add a third check (sorry, I wrote the comment on the wrong line before)

Reply:
We'll have to modify this function anyway when we add support for 3rd party components, and knowing that the function consists on 2 ifs and has full coverage it's safe to refactor